src/Library/demos/Toggle Button/main.blp

```diff
@@ -0,0 +1,82 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "Toggle Button";
+  description: _("Represent active-state visually");
+
+  Box {
+    orientation: vertical;
+
+    Box {
+      orientation: horizontal;
+      halign: center;
+
+      ToggleButton button_first {
+        active: true;
+        icon-name: "audio-volume-muted-symbolic";
+        halign: center;
+        margin-end: 12;
+        margin-bottom: 18;
+      }
+
+      ToggleButton button_second {
+        active: false;
+        icon-name: "list-compact";
+        halign: center;
+        group: button_first;
+        margin-bottom: 18;
+      }
+    }
+
+    Box {
+      orientation: horizontal;
+      halign: center;
+
+      ToggleButton button_third {
+        active: false;
+        icon-name: "photo-camera-symbolic";
+        halign: center;
+        margin-end: 12;
+        margin-bottom: 18;
+      }
+
+      ToggleButton button_fourth {
+        active: false;
+        icon-name: "flashlight-symbolic";
+        halign: center;
+        margin-bottom: 18;
+      }
+    }
+
+    ToggleButton disabled {
+      halign: center;
+      margin-bottom: 18;
+      sensitive: false;
```

Comment:
```suggestion
    ToggleButton {
      halign: center;
      margin-bottom: 18;
```

a disabled example isn't helpful here because it's the same as a regular disabled button